src/unpack/index.ts

```diff
@@ -5,10 +5,11 @@ import { CarIndexedReader, CarReader } from '@ipld/car'
 import { Block } from '@ipld/car/api'
 import { CID } from 'multiformats'
 import exporter from '@vascosantos/ipfs-unixfs-exporter'
-import { UnixFSEntry } from '@vascosantos/ipfs-unixfs-exporter'
+import type { UnixFSEntry } from '@vascosantos/ipfs-unixfs-exporter'
+export type { UnixFSEntry }
 
 // Export unixfs entries from car file
-export async function* unpack (carReader: CarReader|CarIndexedReader, roots?: CID[]): AsyncIterable<UnixFSEntry> {
+export async function* unpack(carReader: CarReader | CarIndexedReader, roots?: CID[]): AsyncIterable<UnixFSEntry> {
```

Comment:
We could simplify this signature too now that the CarReader type is an interface in https://github.com/ipld/js-car/pull/32/files